MCPB_README.md

```
@@ -0,0 +1,172 @@
+5. Confirm the installation
+
+## Usage
+
+Once installed, the extension provides two main tools:
+
+### 1. Geocode Tool
+
+Convert an address or location description to coordinates.
+
+**Example Usage:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)